pkg/monitor/operator.go

```diff
@@ -17,45 +17,6 @@ import (
 	configclientset "github.com/openshift/client-go/config/clientset/versioned"
 )
 
-// condition/Degraded status/True reason/DNSDegraded changed: DNS default is degraded
-func GetOperatorConditionStatus(message string) *configv1.ClusterOperatorStatusCondition {
```

Comment:
bbd9949c834a14 (#26034) moved this to `pkg/monitor/monitorapi`.  :+1: to dropping this old, unused copy.